warehouse/organizations/services.py

```diff
@@ -396,6 +396,10 @@ def add_organization_project(self, organization_id, project_id):
         )
 
         self.db.add(organization_project)
+        self.db.flush() # Flush db so we can address the organization related object
```

Comment:
```suggestion
        self.db.flush() # flush the db now so we can address the organization related object
```

The linter doesn't seem to recognize the comment. Maybe if it's more like in the example?